packages/wonder-blocks-breadcrumb/components/breadcrumb-item.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps} from "@khanacademy/wonder-blocks-core";
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import Link from "@khanacademy/wonder-blocks-link";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * This is the content for the collection of Breadcrumbs
+     */
+    children: string | React.Element<typeof Link>,
+|};
+
+const StyledListItem = addStyle("li");
+
+/**
+ * The BreadcrumbItem represents an individual item in the breadcrumbs list.
+ */
+export default class BreadcrumbItem extends React.Component<Props> {
+    render() {
+        return (
+            <StyledListItem style={styles.item}>
+                {this.props.children}
+            </StyledListItem>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    item: {
+        display: "inline-block",
+        margin: `0 ${Spacing.xxSmall}`,
+        "::after": {
```

Comment:
Hmm... so... I think I'd prefer that we insert the spacers in-between the BreadcrumbItems inside the Breadcrumb component itself, rather than using CSS. It allows us to have greater control over the display of the spacers and also choose to show/hide them from screenreaders (probably hide them - by default Chrome and Firefox don't hide CSS content from screenreaders so having these spacer characters seems not ideal).

Reply:
More details on this: https://tink.uk/accessibility-support-for-css-generated-content/

Reply:
I was going to suggest this as well, but I couldn't think of a good rationale for it.  Thanks for providing some rationale @jeresig.

Reply:
That's a really good point.

I have a possible solution for that:

Separator as a custom <svg> path:
```
<BreadcrumbItem>
  <Link or string goes here />
  <svg/>
<BreadcrumbItem/>
```
What would be the ideal way to build those spacers in WB?

Reading the documentation, it seems I can rely on a custom icon?
```import Icon from "@khanacademy/wonder-blocks-icon";
import type {IconAsset} from "@khanacademy/wonder-blocks-icon";

// Easter egg: what shape am I?
const customIcon: IconAsset = {
    small: "M6.92820 0L13.85640 4L13.85640 12L6.92820 16L0 12L0 4Z",
};
```

Reply:
@jandrade Hmm - so I think this would be better managed inside the Breadcrumbs component itself. The BreadcrumbItem would be left alone but when it comes to rendering the children it would insert the spacer in-between each of the BreadcrumbItems. So the children would essentially be: `[<BreadcrumbItem>, <Spacer>, <BreadcrumbItem>]`. So this wouldn't be something that the user would ever have to specify - although maybe eventually we could provide a way to specify custom ones, maybe something like: `<Breadcrumbs spacer={<Icon/>}> ... </Breadcrumbs>`.

Also: it's a hexagon! 😆

Reply:
hahah oops I didn't check that shape 😊. I changed the that logic to insert the spacer inside the `<BreadcrumbItem />` component (not sure why you don't see these changes here). The main reason for including the spacer inside there is to add the icon inside each `<li>`.

You can see the new implementation here: https://github.com/Khan/wonder-blocks/pull/391/files#diff-741ab7a0a4aa7032d534457c6b866936R60

The output for this new version would be something like this:

```
<ol>
  <li>
    <Link or string />
    <separator />
  </li>
  ...
</ol>
```

If you still think it's better to add the spacer outside of `BreadcrumbsItem`, we would need to move the `<li />` up to the `Breadcrumb` component ([link](https://github.com/Khan/wonder-blocks/pull/391/files#diff-b2b53ffb4b23955fb0995a5c5ac79be2R48)) to preserve this markup/structure.

What do you think?

It's a good idea to add the `spacer` prop to `<Breadcrubs />` :)